.github/workflows/aws.yml

```diff
@@ -43,4 +44,10 @@ jobs:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+    - name: Slack Notification
+      uses: rtCamp/action-slack-notify@v2.0.1
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_USERNAME: GitHubBot
```

Comment:
Do you need to specify a channel or anything here?

Reply:
The channel is inferred from the webhook url